lib/load/merge-configs.js

```diff
@@ -4,7 +4,7 @@ const merge = require('lodash').mergeWith;
 
 module.exports = function(configs) {
     if (!configs || !configs.length) return {};
-    return merge({}, ...configs, (a, b) => {
+    return merge({}, configs, (a, b) => {
```

Comment:
It is not "copy paste from some kind of syntax highlighter". It is a [spread operator](https://developer.mozilla.org/pl/docs/Web/JavaScript/Referencje/Operatory/Spread_operator) to spread a list of configuration objects. Removing them breaks the logic of this library. That is why [Travis build is red](https://travis-ci.org/coditorium/nodejs-read-config/builds/181196980).

Could you show a test of your failing case?